cpp/ql/src/experimental/Security/CWE/CWE-1041/FindWrapperFunctions.ql

```diff
@@ -0,0 +1,140 @@
+/**
+ * @name Find Wrapper Functions
+ * @description --Finding for function calls for which wrapper functions exist.
```

Comment:
Again, I think the _implementation_ of the query is revolved around finding wrapper functions, but the intent of the query is find missed opportunities to call the wrapper function.
```suggestion
 * @description If a wrapper function is defined for a given function, any call to the given function should be via the wrapper function.
```